packages/exchange/src/exchange/providers/anthropic.py

```diff
@@ -164,5 +164,5 @@ def recommended_models() -> tuple[str, str]:
 
     @retry_procedure
     def _post(self, payload: dict) -> httpx.Response:
-        response = self.client.post(ANTHROPIC_HOST, json=payload)
+        response = self.client.post(self.BASE_URL_DEFAULT, json=payload)
```

Comment:
I think this is incorrect as it defers to the default value without considering an override. OTOH, if I make this empty, it fails tests...